doc/nrf/device_guides/working_with_nrf/nrf70/developing/debugging.rst

```diff
@@ -0,0 +1,43 @@
+.. _ug_nrf70_developing_debugging:
+
+Debugging
+#########
+
+.. contents::
+   :local:
+   :depth: 2
+
+This guide describes how to debug applications that use the nRF70 Series companion chips. It covers the following topics:
+
+- Software
+- Firmware
+- Hardware
+- environment
+
+Software
+********
+
+The scope of this section is limited to the nRF70 driver, WPA supplicant and networking stack.
+
+Enable debug features
+=====================
+
+The nRF70 driver, WPA supplicant and networking stack have debug features that can be enabled to get more information about what is happening in the system. The debug features are enabled by default in the development branch of the nRF Connect SDK.
```

Comment:
It would be good if "information about the system" is clarified a bit more.
```suggestion
The nRF70 driver, WPA supplicant, and networking stack have debug features that can be enabled to get more information about the system.
The debug features are enabled by default in the development branch of the |NCS|.
```